### PR TITLE
cs/cas-285 Image Tile Changes

### PR DESCRIPTION
**Pull request by CorySanin**
- Moved display mode above scale in image tile options
- Scale renamed to resolution
- Rename some of the display modes, provide brief explanations of each one
- [Fixed a bug](https://github.com/castlecms/castle.cms/commit/2397772ea9c7273541495582be5307904981a98c) that occurs when resolution is set to original and display is set to one of the cropping modes
- Image tile form shows a preview of the picture's original focal point
- Changed the focal point indicator color to contrast with both light and dark backgrounds (requires recompilation of styles)

### CODE REVIEW

**Comment by CorySanin** [comment]
I made the requested changes on this one

File: castle/cms/static/patterns/focalpointselect.js [hunk 1]
```diff
@@ -85,32 +86,34 @@ define([
 
     render: function(){
       var that = this;
-      if(that.state.hidden){
```
Comment (obct537) [inline castle/cms/static/patterns/focalpointselect.js]: Is this case being handled in some other way?

File: castle/cms/browser/imageinfo.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import json
+from Products.Five.browser import BrowserView
+from castle.cms import utils
+
+class ImageInfoView(BrowserView):
+
+    def __init__(self, context, request):
```
Comment (obct537) [inline castle/cms/browser/imageinfo.py]: Not very important, but for future reference, this declaration isn't necessary. You should only need to override __init__ if you're doing something non-standard

Comment (CorySanin) [reply castle/cms/browser/imageinfo.py]: 👌

File: castle/cms/static/patterns/focalpointselect.js (hunk 1 above)
Comment (CorySanin) [reply castle/cms/static/patterns/focalpointselect.js]: Well the image doesn't actually get hidden anymore... I suppose maybe the state property should be renamed. Though I don't know if that could have unintentional consequences.

Comment (obct537) [reply castle/cms/static/patterns/focalpointselect.js]: Makes sense....I'd recommend just looking for any references to the hidden value, and verify it's not being used anywhere unexpected. Otherways, should be good

Comment (CorySanin) [reply castle/cms/static/patterns/focalpointselect.js]: I didn't see it anywhere else. Gave a more descriptive name.

**Pull request merged**